requirements.txt

```diff
@@ -2,10 +2,13 @@
 # --------------------
 # This file records the packages and requirements needed in order for
 # the library to work as expected. And to run tests.
+aiohttp
```

Comment:
Is `aiohttp` implicitly required by `fsspec`?  Or does it change the behavior of `fsspec` if it's available at runtime?

Reply:
It changes the behavior. `http` protocol did not seem to be supported at all without it:

```python
>>> of = fsspec.open('https://storage.googleapis.com/gef-ckan-public-data/awc-isric-soilgrids/awc.tif.yml')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\dmf\projects\geometamaker\env-test\Lib\site-packages\fsspec\core.py", line 459, in open
    out = open_files(
          ^^^^^^^^^^^
  File "C:\Users\dmf\projects\geometamaker\env-test\Lib\site-packages\fsspec\core.py", line 283, in open_files
    fs, fs_token, paths = get_fs_token_paths(
                          ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\dmf\projects\geometamaker\env-test\Lib\site-packages\fsspec\core.py", line 623, in get_fs_token_paths
    chain = _un_chain(urlpath0, storage_options or {})
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\dmf\projects\geometamaker\env-test\Lib\site-packages\fsspec\core.py", line 332, in _un_chain
    cls = get_filesystem_class(protocol)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\dmf\projects\geometamaker\env-test\Lib\site-packages\fsspec\registry.py", line 238, in get_filesystem_class
    raise ImportError(bit["err"]) from e
ImportError: HTTPFileSystem requires "requests" and "aiohttp" to be installed
```

Reply:
Got it.  It's strange to me that `aiohttp` is listed as an extra requirement in `fsspec`'s `setup.py` but not `requests`.   Oh well!